library/src/morpheus-master/sass/modules/reorderer/_base.scss

```diff
@@ -38,7 +38,6 @@ ul.styled-reorder-list {
     .title {
       display: block;
       float: left;
-      white-space: nowrap;
```

Comment:
this seems like a big change, right @profmikegreene ? Maybe you should create a new CSS class called "title-nowrap" ?